packages/rancher-istio-1.8/charts/values.yaml

```diff
@@ -16,13 +17,17 @@ istiocoredns:
     repository: rancher/mirrored-istio-coredns-plugin
     tag: 0.2-istio-1.1
 
+##Native support for dns added in 1.8
+dns:
+  enabled: false
```

Comment:
Will this stay as default false or eventually move to true ? If it moves to true it might conflict.

Reply:
the default we had/have for the addon is ```istiocoredns:
  enabled: false``` so it made sense to leave the native dns support ```dns:
  enabled: false``` as well. What conflicts are you concerned about specifically?

Reply:
If the istio coredns conflicts with rancher coredns. I don't think it does but if we flip it on we'd need to test.